Cargo.lock

```diff
@@ -210,30 +210,30 @@ checksum = "7079075b41f533b8c61d2a4d073c4676e1f8b249ff94a393b0595db304e0dd87"
 
 [[package]]
 name = "anstyle-parse"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
You don't need to call `cargo update` in your PR. Could you minimise number of changes, please?=)